makerom/user_settings.c

```diff
@@ -559,6 +559,14 @@ int SetArgument(int argc, int i, char *argv[], user_settings *set)
 		set->cia.randomTitleKey = true;
 		return 1;
 	}
+	else if (strcmp(argv[i], "-titlekey") == 0) {
+		if (ParamNum != 1) {
+			PrintArgReqParam(argv[i], 1);
+			return USR_ARG_REQ_PARAM;
+		}
+    set->cia.titleKey = argv[i + 1];
```

Comment:
Can you please change the indentation from spaces to tabs to match the existing code?